torchaudio/csrc/ffmpeg/buffer.cpp

```diff
@@ -5,8 +5,27 @@
 namespace torchaudio {
 namespace ffmpeg {
 
-Buffer::Buffer(AVMediaType type) : media_type(type) {}
+Buffer::Buffer(int frames_par_chunk, int num_chunks)
```

Comment:
should be `per`, not `par`

```suggestion
Buffer::Buffer(int frames_per_chunk, int num_chunks)
```